### PR TITLE
berry: bump to 0.1.8

### PR DESCRIPTION
**Pull request by lowercasename**
## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/berry/version
/var/db/kiss/installed/berry/sources
/var/db/kiss/installed/berry/manifest
/var/db/kiss/installed/berry/depends
/var/db/kiss/installed/berry/checksums
/var/db/kiss/installed/berry/build
/var/db/kiss/installed/berry/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/local/share/man/man1/berryc.1
/usr/local/share/man/man1/berry.1
/usr/local/share/man/man1/
/usr/local/share/man/
/usr/local/share/
/usr/local/bin/berryc
/usr/local/bin/berry
/usr/local/bin/
/usr/local/
/usr/
```

## Existing package

- [X] I am the maintainer of this package.

### CODE REVIEW

**Comment by dilyn-corner** [comment]
[:)](https://github.com/kisslinux/community/pull/1040)

**Comment by lowercasename** [comment]
Thank you! I've done the rebase... but I don't think it worked!

**Comment by dilyn-corner** [comment]
Hm. Perhaps it behaves slightly differently if you do all your work on main. 

You could always just nuke the repository, refork this, and make a branch to bump the version. But that's the spider-in-the-house-so-burn-it-down method.

File: community/brownout/version [hunk 1]
```diff
@@ -1 +1,9 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
 0.4 1
+=======
+git 1
+>>>>>>> b8475701 (brownout: new package at 0.4)
+=======
+0.4 1
+>>>>>>> 5ca5f8cf (Switched sources to tar.gz)
```
Comment (dilyn-corner) [inline community/brownout/version:9]: Your rebasing appears to have been catastrophic. 

[This](https://www.youtube.com/watch?v=kMvLn8WcAII) was on my list of things to watch a while back and you might find it helpful.

Comment (lowercasename) [reply community/brownout/version:9]: Yep, I'm getting it now! I'll close this one and resubmit. :)

**Pull request closed without merging**